.github/workflows/integration-tests.yml

```diff
@@ -15,7 +15,6 @@ jobs:
       max-parallel: 1
       matrix:
         python-version:
-        - '3.6'
```

Comment:
We should run integration test for the lowest supporting version also

Reply:
+1 for @congminh1254